### PR TITLE
Closes #231: Add 'Contributors' page 

### LINKED ISSUE #231: Create "Contributors" page
Would be great to have somewhere to credits contributors on the website and maybe in this repo too.
It could be separated between "Developers", "Translators" etc.

### PR DESCRIPTION
**Pull request by baltpeter**
My draft for a 'Contributors' page. Happy to hear any feedback.

I am not exactly happy with 'Beitragende' as the German translation for 'Contributors'. If anyone has a better idea, please let me know.

### CODE REVIEW

**Comment by zner0L** [comment]
What about 'Mitwirkende'?

File: content/open-source.de.md [hunk 1]
```diff
@@ -85,6 +85,17 @@ Hier findest Du eine Übersicht einiger wichtiger Repositories, die direkt oder
     <a class="button button-primary read-more-button" href="https://github.com/baltpeter/yace">Zum Repository&nbsp;<span class="icon icon-arrow-right"></span></a>
 </article>
 
+<a id="contributors"></a>
+## Beitragende
```
Comment (zner0L) [inline content/open-source.de.md]:
```suggestion
## Mitwirkende
```

Comment (zner0L) [inline content/open-source.de.md]:
```suggestion
## Mitwirkende
```

File: src/i18n/de.json [hunk 2]
```diff
@@ -151,6 +151,8 @@
         "footer-become-a-member-url": "/verein/mitglied-werden",
         "footer-open-source": "Open Source",
         "footer-open-source-url": "/open-source",
+        "footer-contributors": "Beitragende",
```
Comment (zner0L) [inline src/i18n/de.json]:
```suggestion
        "footer-contributors": "Mitwirkende",
```

**Comment by zner0L** [comment]
And another thought: Some authors who contributed articles during the hacktoberfest and added themselves to the `params.toml` file, but not to the `AUTHORS` file. Do you think we should add them ourselves now?

**Comment by baltpeter** [comment]
Mitwirkende is better, let's go with that.

And you're right, we should add the authors as well.

File: AUTHORS [hunk 3]
```diff
@@ -1,6 +1,10 @@
-# This file contains a list of people and organizations who have contributed to this repository (sorted alphabetically). If you have made a contribution, feel free to add your name (or pseudonym, handle etc.). That is of course entirely voluntarily.
+# This file contains a list of people and organizations who have contributed to this repository (sorted alphabetically).
+# If you have made a contribution, feel free to add your name (or pseudonym, handle etc.). That is of course entirely
+# voluntarily.
 # The authors of libraries and other external dependencies we use are listed in the ATTRIBUTION file.
 
 Benjamin Altpeter
 Lewin Sieben
 Lorenz Sieben
```
Comment (NathanBnm) [inline AUTHORS]:
```suggestion
Lorenz Sieben
Nathan Bonnemains
```

Comment (NathanBnm) [reply AUTHORS]: Please add me to the list :smile: (You just have to apply this suggestion).

**Comment by baltpeter** [comment]
Sorry, somehow I lost track of this.

> Maybe it can be great to specify the contributors position and actions.

As I mentioned [before](https://github.com/datenanfragen/website/issues/231#issuecomment-590775672), I am not opposed to that. I am however not sure how we would go about translating that efficiently. Any ideas?

**Comment by NathanBnm** [comment]
@baltpeter what about creating a dedicated "Contributors" page on the website, and then add a link to this page in the README. I think that the AUTHORS find is meant to cite you and @zner0L as authors/founders of the website, but not for contributors.

By doing this is would be easier to translate :smile:

**Comment by baltpeter** [comment]
> what about creating a dedicated "Contributors" page on the website

But that would make it pretty hard for new contributors to add themselves to the list, wouldn't it? They would have to go through all languages and find translations for their 'contribution types' in all languages we support.

File: layouts/shortcodes/authors.html [hunk 4]
```diff
@@ -1 +1 @@
-{{ trim (replaceRE "(?m)^#.+?$" "" (readFile "AUTHORS")) "\n" }}
+{{ replaceRE "([^\n]+)\n*$|\n*(.+\n)\n*" "$1$2" (replaceRE "(?m)^#.+?$" "" (readFile "AUTHORS"))  }}
```
Comment (zner0L) [inline layouts/shortcodes/authors.html:1]: Could you maybe check my regex, before I merge @baltpeter?

**Comment by NathanBnm** [comment]
Just added French version, which is a non-breaking change :smile:

File: layouts/shortcodes/authors.html (hunk 4 above)
Comment (baltpeter) [reply layouts/shortcodes/authors.html:1]: Oh, that is disgusting. :D But, seems ok for what it's supposed to do and if it's necessary… :/

**Comment by baltpeter** [comment]
Seems like there is now a consensus to move forward with this approach (at least for now). I agree. It is better to have some attribution than none. If we want to discuss this further, we can leave #231 open and make more changes later.

In the meantime, this can be merged (as soon as I add the new authors).

@NathanBnm Thanks!

**Pull request merged**